internal/service/appfabric/appfabric_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
```

Comment:
why is this file included?

Reply:
This is a way to test the cases see example on Security lake, and bedrock knowledge base. @ewbankkit will that be the way forward for tests or we should default to the old tests